components/Form/CardForm.js

```diff
@@ -0,0 +1,112 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function CardForm({
+  onAddCard,
+  onCancel,
+  onPopupContentClick,
+  topics,
+  color,
+}) {
+  const [selectedTopic, setSelectedTopic] = useState("");
+  const [question, setQuestion] = useState("");
+  const [answer, setAnswer] = useState("");
+  const [questionError, setQuestionError] = useState(false);
+  const [answerError, setAnswerError] = useState(false);
+
+  const handleTopicChange = (event) => {
+    const value = event.target.value;
+    setSelectedTopic(value);
+  };
+
+  const handleQuestionChange = (event) => {
+    const value = event.target.value;
+    setQuestion(value);
+  };
+
+  const handleAnswerChange = (event) => {
+    const value = event.target.value;
+    setAnswer(value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (question.trim() === "" || answer.trim() === "") {
+      setQuestionError(question.trim() === "");
+      setAnswerError(answer.trim() === "");
+      return;
+    }
+    onAddCard({ selectedTopic, question, answer });
+  };
+
+  return (
+    <PopupContent onClick={onPopupContentClick} color={color}>
+      <form onSubmit={handleSubmit}>
+        <h2>{"Add new Card"}</h2>
+
+        <label htmlFor="topic">Topic</label>
+        <select name="topic" id="topic" onChange={handleTopicChange} required>
+          <option value="" selected disabled hidden>
+            Please select a Topic
+          </option>
+          {topics.map((topic) => (
+            <option key={topic.id} value={topic.title}>
+              {topic.title}
+            </option>
+          ))}
+        </select>
+        <br />
+        <br />
```

Comment:
Don't use `<br />` tags, use styling instead.

Reply:
I used this method temporarily, because I did not want to style my app right now. Is it possible to remove them at a later stage or should I remove them now?

Reply:
You should remove them now.

Reply:
Hint: Your styling don't need to complete at this point.